news/2020-05-15-project-loved-may-2020.md

```diff
@@ -0,0 +1,208 @@
+[Sara'h - DESPACITO ( FRENCH VERSION ) (Speed Up Edit)](https://osu.ppy.sh/beatmapsets/788052#fruits) by [Vincs](https://osu.ppy.sh/users/2026299)  
+263 BPM, 2:13 | 6.33★  
+*description written by [Wesley](https://osu.ppy.sh/users/2407265)*
+
+We're back again with a map from Vincs, a French cover of the well-known song *Despacito*. The map was created as a warmup for the osu!catch World Cup 2018 for the French team. The map was used quite a bit in multiplayer lobbies back around the year and hasn't seen much sunlight lately. The song starts out slow, but it quickly builds up so that you have to move left and right all over the place, getting close to no breaks to slow down at all. In order to enjoy the full experience, make sure to turn on the hitsounds and enjoy the beauty Vincs has created.
+
+[![](/wiki/shared/news/2020-05-15-project-loved-may-2020/catch/3-fire.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[BTS - FIRE](https://osu.ppy.sh/beatmapsets/731684#fruits) by [Spectator](https://osu.ppy.sh/users/702598) and [Ollapeu](https://osu.ppy.sh/users/2064505)  
+100 BPM, 3:22 | 5.94★, 4.79★  
+*description written by [Sartan](https://osu.ppy.sh/users/4100941)*
+
+Coming in with the most fire song of this osu!catch round of Loved (hihi haha) we have another entry from our dear veteran mapper Spectator. This set features only two difficulties, but with the song being 3:22 long, there is still quite a lot to be seen. In this set we have a surprisingly nicely mapped rain by Ollapeu, somebody that has not yet been featured in Loved. There are not real weird gimmicks to be found in this set, it is very purely mapped, but that does not mean it can't pose a challenge even to the better players in our community when combined with mods such as HardRock or even DoubleTime on either of these difficulties. So go fire up your game (I did it again! Hihihi hahaha) and check it out!
+
+[![](/wiki/shared/news/2020-05-15-project-loved-may-2020/catch/4-come-back-home.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[2NE1 - COME BACK HOME](https://osu.ppy.sh/beatmapsets/294440#fruits) by [119410501](https://osu.ppy.sh/users/803484)  
+150 BPM, 3:47 | 8.18★  
+*description written by [Tenshichan](https://osu.ppy.sh/users/1101600)*
+
+119410501 is known as a very skilled player in the catch community, yet his maps are not known nowadays and haven't been receiving much attention, which I think is a shame, because he made some fun maps that certainly do deserve more attention than they get. One of those maps is *COME BACK HOME*. It is a very hard map with unorthodox patterns and some of them feel close to impossible to catch. It's easy to miss an entire pattern and die because of the drain. Let alone getting a pass is really difficult. But this challenging aspect is what makes the map fun and increases its replay value. Meaning if you are looking for a challenge that might take you a while, then make sure to give this map a try!
```

Comment:
```suggestion
119410501 is known as a very skilled player in the catch community, yet his maps are not known nowadays and haven't been receiving much attention, which I think is a shame, because he made some fun maps that certainly do deserve more attention than they get. One of those maps is *COME BACK HOME*. 

A very hard map with unorthodox patterns bordering on the impossible, it is easy to miss an entire pattern and die because of the drain, let alone getting a pass!. But this challenging aspect is what makes the map so fun and increases its replay value. If you are looking for a challenge that might take you a while, then make sure to give this map a try!
```